src/views/Correction.vue

```diff
@@ -102,6 +102,7 @@
                 </p>
               </header>
               <v-card flat>
+                <div :class="{ 'invalid-section': !detailCommentValid && showErrors}">
                 <DetailComment
```

Comment:
You probably don't need the div around this -- you can add the class directly to this component itself, like any other HTML element.

Ditto line 132 below.